doc/tutorials/01-lennard_jones/01-lennard_jones.ipynb

```diff
@@ -700,7 +700,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.6.9"
   }
```

Comment:
This change happens when you accidentally run the tutorials in ipypresso in the source directory instead of the build directory. Jupyter will autosave the notebook and update the version number. This change can be safely reverted.